rust/definitions/src/history.rs

```diff
@@ -155,24 +146,14 @@ impl NetworkSpecsExport {
             signed_by: signed_by.to_owned(),
         }
     }
-
-    /// Print json with [`NetworkSpecsExport`] information for user interface
-    #[cfg(feature = "signer")]
-    pub fn show(&self) -> String {
-        format!(
-            "{},\"signed_by\":{}",
-            &self.specs_to_send.show(),
-            export_complex_single(&self.signed_by, |a| a.show_card())
-        )
-    }
 }
 
 /// Event content for setting network verifier
 #[derive(Debug, Decode, Encode, PartialEq, Clone)]
 pub struct NetworkVerifierDisplay {
```

Comment:
Shows network genesis hash instead of verifier public key in log message.

Reply:
i think fixed now